data/hai/hai reveal 0 prologue.txt

```diff
@@ -0,0 +1,734 @@
+			label briefing
+			`	Alondo leads you to a meeting room that feels large for the number of people present: it looks like a lecture hall that could easily fit a hundred people, but there are only six here. Aside from Alondo and yourself, there is a shorter man in what you recognize as a Republic diplomatic uniform messing with a holoprojector that doesn't appear to be working, a taller man in a suit sitting next to a woman in a formal dress in the front row, and an older looking Hai woman sitting alone at the end of a row of seats.`
+			`	"Glad you finally decided to show up, Alondo," says the man in the front.`
+			`	"And you brought a guest," says the Hai.`
+			`	"I did," Alondo responds. "This is Captain <first> <last>. Member of the Free Worlds Council during the war. Helped repel the Pug invasion. A capable pilot. Am I missing anything?"`
+			choice
+				`	"That sounds like everything."`
+				`	"Just that I'm good looking."`
+			`	"There we go!" says the man who was fiddling with the holoprojector as it whirs to life. "No time for introductions. We're late enough as is. Everyone, please sit down."`
+			`	The holoprojector displays a series of graphs. You're no expert in data analysis, but a highlighted "chance of total crisis" at the bottom ranging from 61% to 96% does not look promising.`
+			`	"Please turn your attention to this chart here," he says while pointing to a bar graph. "We've continued to track the number of new arrivals to Hai territory. As noted on the left, the number of new monthly arrivals numbered in only the low hundreds prior to the leaks being broadcast. Within only a short time after, that number quickly rose to the thousands. Mitigation efforts by the Navy to dissuade civilian captains from discovering the wormhole in Ultima Thule by circulating reports of increased pirate activity have since suppressed this number somewhat, but the rate of new arrivals is still well above what it once was.`
+			`	"It has been noted that the character of these new arrivals is also different. Where once a majority of new arrivals were lone explorers or lost captains, they are now increasingly captains following breadcrumbs while looking for the Hai, or even captains bringing large groups of 'tourists' with the promise of witnessing the aliens from the news reports. Further research will need to be done to determine the threat that these new arrivals pose to further escalating the situation."`
+			choice
+				`	"What's the problem with new arrivals?"`
+				`	"Do we not have a way of blocking captains from entering the wormhole?"`
+					goto block
+			`	"We do not have the means of handling a large influx of humans," says the Hai. "For decades, we have dealt with a small trickle of humans arriving through the wormhole. We have grown acustomed to it. But if that trickle were to suddenly become a downpour, then our services would quickly become overrun."`
```

Comment:
When I got to Hai space, I didn't notice any services. Is there something I'm missing?

Reply:
You were presumably not disembarking from a ship with the intent of settling there.

Reply:
Just any general services that you might expect of a city or a spaceport to have in supporting a population, whether that be permanent or temporary. If you have tens or even hundreds of thousands of extra people that you weren't expecting to have to deal with, even if they're just tourists who are passing through, then refueling, sanitation, food services, housing, etc. will all be strained and could break.